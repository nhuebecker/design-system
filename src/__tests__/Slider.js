import React from 'react'
import renderer from 'react-test-renderer'
import { Slider, theme } from '..'

describe('Slider', () => {
  test('renders', () => {
    const json = renderer.create(<Slider id="renders" />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
