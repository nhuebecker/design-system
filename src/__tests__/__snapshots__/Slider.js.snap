// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider renders 1`] = `
.c0 {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: block;
  border-radius: 9999px;
  height: 4px;
  cursor: pointer;
  width: 100%;
  background-color: #dde1e4;
  color: #e42d42;
}

.c0::-webkit-slider-thumb {
  width: 16px;
  height: 16px;
  background-color: currentColor;
  border: 0;
  border-radius: 9999px;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

<input
  className="c0"
  color="primary"
  id="renders"
  type="range"
/>
`;
